sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -1872,8 +1872,8 @@ ESIMD_INLINE SYCL_ESIMD_FUNCTION void lsc_prefetch_2d(
   constexpr uint8_t numSrc0 = 0x1;
   constexpr uint8_t numSrc1 = 0x0;
   constexpr uint8_t numDst = (N * sizeof(T)) / 64;
-  raw_sends_store(oldDst, payload.get_raw_data(), exDesc, desc, execSize, sfid,
-                  numSrc0, numDst);
+  raw_send_load(oldDst, payload.get_raw_data(), exDesc, desc, execSize, sfid,
```

Comment:
This function is lowered to __esimd_raw_send2(), but it should be lowered to __esimd_raw_send2_no_result()